tests/test_data_types.py

```diff
@@ -42,3 +39,13 @@ def test_that_y_has_to_be_positive(self):
     def test_that_x_and_y_have_to_be_integers(self):
         with self.assertRaises(TypeError):
             Stride(1.2, 1.5)
+
+
+class TestGreyscaleImage(unittest.TestCase):
```

Comment:
This looks like copy/paste of TestCase for class Patch. I its just silly mistake?